manifests/prospector.pp

```diff
@@ -31,6 +31,17 @@
   }
 
   case $::kernel {
+    'FreeBSD' : {
+      file { "filebeat-${name}":
+        ensure  => $ensure,
+        path    => "${filebeat::config_dir}/${name}.yml",
+        owner   => 'root',
+        group   => 'wheel',
```

Comment:
It looks like the only difference here is the group. Does it make more sense to parameterize the group or have a separate resource?